tests/integration/update_cluster/shared_vpc_ipv6/kubernetes.tf

```diff
@@ -22,7 +22,6 @@ locals {
   vpc_cidr_block                    = data.aws_vpc.minimal-ipv6-example-com.cidr_block
   vpc_id                            = "vpc-12345678"
   vpc_ipv6_cidr_block               = data.aws_vpc.minimal-ipv6-example-com.ipv6_cidr_block
```

Comment:
This integration test case uses ipv6 so we probably shouldn't be removing it here. In general we may not be able to know if a "shared VPC" has ipv6 enabled or not